### PR TITLE
Second trial to create a pull request

### PR DESCRIPTION
**Pull request by gal42**
Created a file with my favorite movie

### CODE REVIEW

**Review by rachelkang — CHANGES_REQUESTED** [review]
Awesome job!!

One small suggestion I have - could you move this file so that it's inside the GalExample directory? And then delete the trail2 directory since it will be empty

File: trial2/text2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+My favorite movie is Big Hero 6
```
Comment (rachelkang) [inline trial2/text2.txt]: A great movie!!

Comment (gal42) [reply trial2/text2.txt]: Hi Rachel! I this I successfully moved the file and deleted the directory!

Comment (rachelkang) [reply trial2/text2.txt]: Awesome - great job!

**Review by rachelkang — APPROVED** [review]
LGTM!

One more nit suggestion for the future - remember to rename the file so that it is relevant to the content it holds. For the purpose of this exericse, however, this is great!

**Review by tj-devel709 — APPROVED** [review]

**Pull request merged**